cmd/buildkitd/main.go

```diff
@@ -373,7 +373,7 @@ func applyMainFlags(c *cli.Context, cfg *config.Config, md *toml.MetaData) error
 	if tlskey := c.String("tlskey"); tlskey != "" {
 		cfg.GRPC.TLS.Key = tlskey
 	}
-	if tlsca := c.String("tlsca"); tlsca != "" {
+	if tlsca := c.String("tlscacert"); tlsca != "" {
```

Comment:
`--tlsca` was not effective: `Incorrect Usage. flag provided but not defined: -tlsca`

So this change won't break any existing user